resources/js/components/GraphqlMutation.vue

```diff
@@ -199,5 +199,10 @@ export default {
             })
         },
     },
+    computed: {
+        running() {
+            return this.mutating
+        },
```

Comment:
Maybe better to turn this around; change it to `running` everywhere for consistency and keep a `mutating` computed to keep it backwards compatible? So when we like to drop this "backwards compatible fix" we can just remove the computed.

Reply:
Agreed, i've updated the PR